ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1091,6 +1205,62 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+            test_name: windows-containerd-e2e-ovs
+            node: 'antrea-windows-containerd-testbed'
+            description: 'This is the {test_name} test for {name}.'
+            branches:
+                - ${{sha1}}
+            builders:
+                - builder-e2e-win-containerd:
```

Comment:
need to update this builder accordingly to pass testcase having `containerd-with-ovs` keyword.